src/Components/ImageOrVideo.scss

```diff
@@ -15,6 +15,14 @@
   border: none;
   cursor: pointer;
   transition: all ease-in-out 0.4s;
+  opacity: 0;
+  pointer-events: none;
+
+  &.is-paused {
+    opacity: 1;
+    pointer-events: auto;
```

Comment:
* [ ] Still interferes with user input for people with reduced motion preference. How about only setting this style if motion is preferred? It would no longer animate on hover in the "reduced motion" case, but this is actually good.